<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopConfigDialog</class>
 <widget class="QDialog" name="DesktopConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>259</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Session Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_codec">
     <property name="text">
      <string>Codec:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_codec"/>
   </item>
   <item>
    <widget class="QLabel" name="label_color_depth">
     <property name="text">
      <string>Color depth:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_color_depth"/>
   </item>
   <item>
    <widget class="QLabel" name="label_compression_ratio">
     <property name="text">
      <string>Compression ratio:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_fast">
       <property name="text">
        <string>Fast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_compression_ratio">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_best">
       <property name="text">
        <string>Best</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_update_interval">
       <property name="text">
        <string>Screen update interval (ms):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_update_interval">
       <property name="minimum">
        <number>15</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_cursor_shape">
     <property name="text">
      <string>Enable cursor shape</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_clipboard">
     <property name="text">
      <string>Enable clipboard</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>DesktopConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>DesktopConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
